song_match/effect/effects/game_won.py

```diff
@@ -0,0 +1,26 @@
+from cozmo.anim import Triggers
+
+from song_match.effect.effect import Effect
+
+class GameWonEffect(Effect):
```

Comment:
We should rename this to `GameOverEffect`.

`GameWonEffect` implies they both win, rather than Cozmo or the player wins.

Reply:
Sounds good.